CHANGELOG.rst

```diff
@@ -4,6 +4,15 @@ Change Log
 5.x
 ---
 
+5.1.0 (not yet released)
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Added*
+
+* ``mpcd.update.ReverseNonequilibriumShearFlow``
+  (`#1983 <https://github.com/glotzerlab/hoomd-blue/pull/1983>`__).
+
```

Comment:
Please merge the base branch in and resolve conflicts so that `5.1.0 (not yet released)` is listed above `5.0.1`.